.github/workflows/cargo-test.yaml

```diff
@@ -14,5 +14,22 @@ jobs:
         with:
           profile: minimal
           toolchain: 1.87.0
+
+      - name: Install solana cli
+        run: |
+          sh -c "$(curl -sSfL https://release.anza.xyz/v2.2.20/install)"
+          echo "$HOME/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
+
+      - name: Install anchor cli
```

Comment:
nit: you could save some time here by not installing anchor and building with `cargo-build-sbf`